src/Nordea/Components/DatePicker.elm

```diff
@@ -0,0 +1,374 @@
+module Nordea.Components.DatePicker exposing (OptionalConfig(..), view)
+
+import Css
+    exposing
+        ( absolute
+        , alignItems
+        , alignSelf
+        , backgroundColor
+        , border3
+        , borderBox
+        , borderRadius
+        , borderRadius4
+        , boxSizing
+        , center
+        , column
+        , cursor
+        , display
+        , displayFlex
+        , flexDirection
+        , flexEnd
+        , focus
+        , fontSize
+        , height
+        , hover
+        , int
+        , justifyContent
+        , none
+        , outline
+        , padding
+        , padding2
+        , paddingRight
+        , pct
+        , pointer
+        , pointerEvents
+        , position
+        , relative
+        , rem
+        , right
+        , row
+        , solid
+        , spaceBetween
+        , top
+        , width
+        , zIndex
+        )
+import Date exposing (Date)
+import Html.Styled as Html exposing (Attribute, Html, input, styled)
+import Html.Styled.Attributes exposing (css, placeholder, tabindex, value)
+import Html.Styled.Events as Events exposing (onBlur, onClick, onInput)
+import Json.Decode as Decode
+import List.Extra as List exposing (groupsOf)
+import Maybe.Extra exposing (toList)
+import Nordea.Components.Button as Button
+import Nordea.Components.OnClickOutsideSupport as OnClickOutsideSupport
+import Nordea.Components.Text as Text
+import Nordea.Css exposing (gap)
+import Nordea.Html as Html
+import Nordea.Html.Events as Events
+import Nordea.Resources.Colors as Colors
+import Nordea.Resources.Icons as Icons
+import Nordea.Themes as Themes
+import Time exposing (Month, Weekday)
+
+
+type alias Config msg =
+    { hasFocus : Bool
+    , onFocus : Bool -> msg
+    , input : String
+    , onInput : String -> msg
+    , today : Date
+    , onSelect : String -> msg
```

Comment:
Could this return a Date directly?

Reply:
Yes, we can return date and remove the formatter.